app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -198,6 +198,10 @@ def curation_concern_from_search_results
     def search_result_document(search_params)
       _, document_list = search_results(search_params)
       raise CanCan::AccessDenied.new(nil, :show) if document_list.empty?
+      if current_user.blank? && document_list.present? && document_list.first["suppressed_bsi"]
```

Comment:
I was using `blank?` to display the message only for anonymous users, should I use `unless` instead of blank?
```
unless current_user
  document_list.present? && document_list.first["suppressed_bsi"]
end
```